framework/src/main/java/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -1357,7 +1358,20 @@ protected V getWidenedValue(TypeMirror type, V annotatedValue) {
     public TransferResult<V, S> visitStringConversion(
             StringConversionNode n, TransferInput<V, S> p) {
         TransferResult<V, S> result = super.visitStringConversion(n, p);
-        result.setResultValue(p.getValueOfSubNode(n.getOperand()));
+        TypeMirror strType = n.getType();
```

Comment:
You could move this declaration down to just before `resultAnnos`, putting it closer to its first use.